Makefile

```diff
@@ -1,15 +1,18 @@
 PWD ?= $$(pwd)
-USERID ?= $$(id -u)
```

Comment:
Why not keep this? This is used to chown the files automatically in the docker image entrypoint.

Reply:
Really? It breaks on my Linux machine. I'll submit an issue for it.